helm/cas-bciers/values-test.yaml

```diff
@@ -75,7 +75,7 @@ reportingFrontend:
       memory: 128Mi
 
 complianceFrontend:
-  replicaCount: 1
+  replicaCount: 2
```

Comment:
Should we bump this one now? I'm just testing the Registration, dashboard, and administration apps.